validator/src/cli.rs

```diff
@@ -1723,6 +1735,7 @@ impl DefaultArgs {
             exit_max_delinquent_stake: "5".to_string(),
             wait_for_restart_window_min_idle_time: "10".to_string(),
             wait_for_restart_window_max_delinquent_stake: "5".to_string(),
+            banking_trace_size: DEFAULT_BANKING_TRACE_SIZE.to_string(),
```

Comment:
Won't this enable it by default?

Reply:
this won't. see db4732c1efec119e47e23e880bf2c7b8e1742d04